test/integration/prefs.js

```diff
@@ -0,0 +1,78 @@
+const chai = require('chai');
+const sinon = require('sinon');
+const sinonChai = require('sinon-chai');
+const chalk = require('chalk');
+const inquirer = require('inquirer');
+const Table = require('cli-table');
+const controller = require('../../lib/controller');
+const preferenceManager = require('../../lib/utils/preference-manager');
+
+chai.use(sinonChai);
+chai.should();
+
+describe('Integration test for init command', () => {
```

Comment:
May be the wrong phrase. This is supposed to be integration test for prefs command. Due to the clash, only one init command shows up in the [build log](https://travis-ci.org/AutolabJS/autolabcli/builds/355524704)